less/index.less

```diff
@@ -65,112 +65,112 @@ h1, h2, h3, h4, h5 {
 }
 
 .container {
-    width: 880px;
+    width: 800px + 80;
     margin: auto;
 }
 
 /* Header Styles */
 
 header {
-    width: 100%;
+    width: 50% * 2;
     display: flex;
-}
 
-header nav {
-    margin-top: 40px; 
-    width: 610px;
-    justify-content: space-between;
-    display: flex;
-}
+    nav {
+        margin-top: 80px / 2; 
+        width: 600px + 10;
+        justify-content: space-between;
+        display: flex;
+    }
 
-header nav a {
-    text-decoration: none;
-    color: #b0b0b0;
-}
+    nav a {
+        text-decoration: none;
+        color: #b0b0b0;
+    }
 
-header nav a:hover {
-    text-decoration: underline;
-    color: black;
-}
+    nav a:hover {
+        text-decoration: underline;
+        color: black;
+    }
 
-header .logo {
-    align-self: flex-end;
-    margin: 28px 0 0 91px;
+    .logo {
+        align-self: flex-end;
+        margin: 14px * 2 0 0 92px - 1;
+    }
 }
 
 /* Call To Action (CTA) Styles */
 
 .cta {
     display: flex;
-    margin-top: 72px;
+    margin-top: 9px * 8;
     justify-content: flex-end;
-}
 
-.cta .cta-text {
-    font-size: 72px;
+    .cta-text {
+    font-size: 70px + 2;
     text-align: center;
     letter-spacing: 5px;
     display: flex;
     justify-content: center;
     flex-flow: column;
     align-items: center;
-    margin: 0 82px 0 0;
-}
+    margin: 0 80px + 2 0 0;
+    }
 
-.cta .cta-text button {
+    button {
     border: 1px double black;
-    font-size: 16px;
+    font-size: 4px * 4;
     background: white;
-    width: 170px;
-    height: 32px;
-    margin-top: 10px;
+    width: 100px + 70;
+    height: 8px * 4;
+    margin-top: 5px + 5;
     cursor: pointer;
-}
+    }
 
-.cta .cta-text button:hover {
-    background: black;
-    color: white;
+    button:hover {
+        background: black;
+        color: white;
+    }
 }
 
 /* Main Content Styles */
 
 .main-content {
    border-top: 2px solid black; 
    border-bottom: 2px solid black;
-   margin: 75px 0 35px 0;
-   padding: 35px 0;
-   font-size: 16px;
-   line-height: 18px;
-}
+   margin: 5px * 15 0 5px * 7 0;
+   padding: 5px * 7 0;
+   font-size: 4px * 4;
+   line-height: 9px * 2;
 
-.main-content .top-content {
-    display: flex;
-    justify-content: space-around;
-}
+    .top-content {
```

Comment:
So you're doing a lot of sandboxing with the LESS, and that's great, so now try to start refactoring your code to utilize LESS in a very effective, economic way. Take out the math operators, try to find some useful mixins, etc. For example, right here, `top-content` and `bottom-content` are both using `display: flex` and `justify-content: space-around`. So why not make a mixin for that called something like `.flex-space-around()` and then apply that to top and bottom content instead?